src/screens/Home.tsx

```diff
@@ -16,7 +16,10 @@ export default function Home() {
 
     return (
         <View style={styles.container}>
+            <Pressable
+                onPress={() => navigation.navigate('AboutPage')}>
             <Text style={styles.title}>HotSoup</Text>
```

Comment:
Please indent this line properly: `<Text` style={styles.title}>HotSoup</Text>`

@rweitzman I think this code can be merged into dev as is, and Jaideep can merge it to his. However, we might want to review the design decision to make the title a navigable button.